test/e2e/plugin_lifecycle/plugin_lifecycle_helper.go

```diff
@@ -37,3 +37,10 @@ func SearchAllPluginGroups(tf *framework.Framework, opts ...framework.E2EOption)
 	gomega.Expect(err).To(gomega.BeNil(), "should not get any error for plugin search")
 	return pluginGroups
 }
+
+// ListInstalledPlugins runs the plugin list command and returns all the plugins from the list output
```

Comment:
nit: It would be best to relocate this to the framework_helper.go file and not validate error with Expect. Instead, it is better to return the error and let user handle to validate error.

Reply:
I removed this helper function and used `installedPluginList, err := tf.PluginCmd.ListInstalledPlugins()` function directly as it seemed unnecessary to add this new function.